pkg/storkctl/clusterpair.go

```diff
@@ -522,100 +450,189 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 						storkCreatedAnnotation: "true",
 					},
 				},
-				Data: credentialData,
-				Type: v1.SecretTypeOpaque,
-			}
-			_, err = core.Instance().CreateSecret(secret)
-			if err != nil {
-				util.CheckErr(err)
-				return
-			}
-
-			backupLocation.Location.SecretConfig = clusterPairName
-			var annotations = make(map[string]string)
-			annotations[skipResourceAnnotation] = "true"
-			_, err = storkops.Instance().CreateBackupLocation(backupLocation)
-			if err != nil {
-				err := fmt.Errorf("unable to create ObjectstoreLocation in source. Err: %v", err)
-				util.CheckErr(err)
-				return
-			}
-			printMsg(fmt.Sprintf("ObjectstoreLocation %v created on source cluster in namespace %v\n", clusterPairName, cmdFactory.GetNamespace()), ioStreams.Out)
-
-			printMsg("Creating a cluster pair. Direction: Source -> Destination", ioStreams.Out)
-			srcClusterPair.Spec.Options[storkv1.BackupLocationResourceName] = backupLocation.Name
-			_, err = storkops.Instance().CreateClusterPair(srcClusterPair)
-			if err != nil {
-				util.CheckErr(err)
-				return
+				Location: storkv1.BackupLocationItem{},
 			}
-			printMsg(fmt.Sprintf("ClusterPair %s created successfully. Direction Source -> Destination\n", clusterPairName), ioStreams.Out)
 
-			if unidirectional {
-				return
-			}
+			if len(backupLocationName) > 0 {
+				err = verifyBackupLocationExistence(sFile, backupLocationName, cmdFactory.GetNamespace())
+				if err != nil {
+					util.CheckErr(fmt.Errorf("input backuplocation %s existence check failed with error in source cluster: %v", backupLocationName, err))
```

Comment:
nit..backupLocation or BackupLocation(just like ObjectstoreLocation in other places)

Reply:
changed to objectstorelocation